util/cron/common.bash

```diff
@@ -110,18 +110,3 @@ fi
 if [ -z "$CHPL_TEST_COMP_PERF_DIR" ]; then
     export CHPL_TEST_COMP_PERF_DIR=$logdir_prefix/NightlyPerformance/$(hostname -s)
 fi
-
```

Comment:
Loosely related. While testing this (and other recent changes) on SLES 12 server, I found out that Cray I.T. has changed their recommended modules setup. I hope to gradually squeeze the module setups out of the Chapel test scripts.  Not the module loads; only "source <some/path>/modules/bash", "module use <some/path>", etc, which are host-specific.  I want those to come from some host-specific script maintained outside of Chapel. (.bashrc for example)